src/windowsFirefox.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------
```

Comment:
Run `npm run fmt` for formatting please :)

Reply:
I think I ran `npm run fmt` before and it ended up showing around 20 files changed? I couldn't see any changes in the diff viewer, so I'm afraid that they're line ending changes. Should I still commit the files?

Reply:
I just ran format on anything I changed